.circleci/config.yml

```diff
@@ -0,0 +1,19 @@
+version: 2.1
+
+orbs:
+  orb-tools: circleci/orb-tools@9.0.0
+
+workflows:
+  validate_publish-dev:
+    jobs:
+      - orb-tools/lint
+      - orb-tools/pack:
+          requires:
+            - orb-tools/lint
```

Comment:
Can't we run `lint` and `pack` in parallel? I think we can move this constraint to the publish job.

Reply:
Sure